snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -95,15 +95,15 @@ public void CreateBucket()
         {
             var projectId = _fixture.ProjectId;
 
-            // <Buckets.Insert>
+            // <Service.Buckets.Insert>
```

Comment:
But they'd be snippets on a class which we don't have the documentation for in this repository... there's just nowhere for them to go. If we start including *all* the code in the docs (as per my experiment at http://jskeet.github.io/gcloud-dotnet) then we could potentially do it, but I'll need to think about exactly how.